<ui version="4.0" >
 <class>NMGTraceWidgetBase</class>
 <widget class="QWidget" name="NMGTraceWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Trace</string>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>356</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton" >
       <property name="text" >
        <string>St&amp;op</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton" >
       <property name="text" >
        <string>St&amp;art</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="hostsView" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Host</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Alias</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Time</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Hosts</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="sourceLabel" >
            <property name="minimumSize" >
             <size>
              <width>70</width>
              <height>14</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>70</width>
              <height>14</height>
             </size>
            </property>
            <property name="text" >
             <string>Source</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="NMGHostInformationComboWidget" native="1" name="sourceBox" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="destinationLabel" >
            <property name="minimumSize" >
             <size>
              <width>70</width>
              <height>14</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>70</width>
              <height>14</height>
             </size>
            </property>
            <property name="text" >
             <string>Destination</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="NMGHostInformationComboWidget" native="1" name="destinationBox" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>NMGHostInformationComboWidget</class>
   <extends>QWidget</extends>
   <header location="global" >nmghostinformationcombowidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >nmghostinformationcombowidget.h</include>
  <include location="local" >nmghostinformationcombowidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>NMGTraceWidgetBase</receiver>
   <slot>startTrace()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>NMGTraceWidgetBase</receiver>
   <slot>stopTrace()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
